articles/fleetctl.md

```diff
@@ -15,9 +15,11 @@ Download and install [Node.js](https://nodejs.org/en).
 Install fleetctl with npm (included in Node.js).
 
 ```sh
-sudo npm install -g fleetctl
+npm install -g fleetctl
 ```
 
+Some configurations may require you to use `sudo` to install fleetctl. 
```

Comment:
@mason-buettner what configurations require `sudo`? I think we want to be explicit.

Reply:
Can we be even more explicit? Imagine someone new to npm.

How do I know if my account is unable to run npm install? Do I see an error message related to needing sudo permissions?

I think we want the copy to be something like this:

"If you see a 'error goes here' error, run sudo npm install -g fleetctl"